types/gloas/block.yaml

```diff
@@ -0,0 +1,77 @@
+Gloas:
+  BeaconBlockBodyCommon:
+    # An abstract object to collect the common fields between the BeaconBlockBody and the BlindedBeaconBlockBody objects
```

Comment:
> common fields between the BeaconBlockBody and the BlindedBeaconBlockBody 

this is no longer true, we can just define all fields in `BeaconBlockBody` for gloas